docs/source/concepts/KafkaLiveStreams.rst

```diff
@@ -43,5 +43,58 @@ This following python script shows how this listener is used:
       Listener='KafkaHistoListener', Address='kafkabroker:address', AccumulationMethod='Replace',
       RunTransitionBehavior='Stop', OutputWorkspace='testout')
 
+Topic Configuration
+###################
+
+The topics that the listener will subscribe to are defined for a particular instrument in Mantid's "Facilities.xml" file. Here is the entry for the V20 beamline for example:
+
+.. code-block:: xml
+
+    <instrument name="V20" shortname="V20" >
+      <zeropadding size="8" />
+      <technique>ESS Test Beamline</technique>
+      <livedata default="event">
+        <connection name="event" address="192.168.1.80:9092" listener="KafkaEventListener" />
+     <topic name="V20_choppers" type="chopper" />
+     <topic name="V20_motion" type="sample" />
+     <topic name="V20_runInfo" type="run" />
+     <topic name="denex_detector" type="event" />
+     <topic name="monitor" type="monitor" />
+      </livedata>
+    </instrument>
+
+.. list-table:: Topic configuration
+   :widths: 15 15 30 30
+   :header-rows: 1
+
+   * - Type
+     - Schema (see https://github.com/ess-dmsc/streaming-data-types)
+     - Required
+     - Description
+   * - chopper
+     - tdct
+     - No (topic doesn't have to exist)
+     - Neutron chopper top-dead-centre timestamps
+   * - sample
+     - f142
+     - Yes (but topic can be empty)
+     - Used to populate workspace logs. "sample" from "sample environment" which is the typical source of these data.
+   * - run
+     - pl72, 6s4t
+     - Yes (there must be a pl72 run start message on the topic for the listener to start successfully)
+     - Row 1, column 3
+   * - event
+     - ev42
+     - Yes (but topic can be empty)
+     - Detection event data
+   * - monitor
+     - ev42
+     - Yes (but topic can be empty)
+     - Detection event data from monitors (single pixel detectors). This just allows using a separate topic for these data, alternatively they can be published to the "event" topic with other data from other detectors.
+
+Note, there must be a run start message (schema pl72) available in the "run" topic for the listener to start. If the "nexus_structure" field of this message contains geometry information in NeXus format (NXoff_geometry or NXcylindrical_geometry) then Mantid will parse this to get the instrument geometry and expected detector ids etc. Otherwise it uses the "instrument_name" to look up a Mantid Instrument Definition File (IDF) for the instrument.
```

Comment:
Minor point: Regarding the "lookup" for instrument information. These are basically the same rules that `LoadInstrument` uses. You might wan to cross-reference.